Formula/mesa.rb

```diff
@@ -3,12 +3,19 @@ class Mesa < Formula
 
   desc "Graphics Library"
   homepage "https://www.mesa3d.org/"
-  url "https://mesa.freedesktop.org/archive/mesa-22.0.5.tar.xz"
-  sha256 "5ee2dc06eff19e19b2867f12eb0db0905c9691c07974f6253f2f1443df4c7a35"
   license "MIT"
-  revision 1
   head "https://gitlab.freedesktop.org/mesa/mesa.git", branch: "main"
 
+  stable do
+    url "https://mesa.freedesktop.org/archive/mesa-22.1.4.tar.xz"
+    sha256 "670d8cbe8b72902a45ea2da68a9da4dc4a5d99c5953a926177adbce1b1640b76"
+
+    patch do
+      url "https://raw.githubusercontent.com/yurikoles/formula-patches/1a1d7d6/mesa/fix-build-on-macOS.patch"
```

Comment:
Is there an upstream commit we can use instead?

Reply:
It doesn't apply cleanly on `22.1.4` tag.